podcasts/New Search/Helpers/SearchAnalyticsHelper.swift

```diff
@@ -0,0 +1,80 @@
+import SwiftUI
+import PocketCastsServer
+
+class SearchAnalyticsHelper: ObservableObject {
+    let source: AnalyticsSource
+
+    init(source: AnalyticsSource) {
+        self.source = source
+    }
+
+    // MARK: - Search
+
+    func trackShown() {
+        Analytics.track(.searchShown, properties: ["source": source])
+    }
+
+    func trackSearchPerformed() {
```

Comment:
I agree.

However, this is how it works with the current search. So I decided to keep the behavior. I know we can improve that but maybe it would be a "nice to have"?